packages/build/config/tsconfig.common.json

```diff
@@ -6,10 +6,10 @@
     "noImplicitAny": true,
     "strictNullChecks": true,
 
-    "lib": ["es2017", "dom"],
+    "lib": ["es2018", "dom"],
     "module": "commonjs",
     "moduleResolution": "node",
-    "target": "es6",
+    "target": "es2017",
```

Comment:
Since we no longer support Node.js 6.x, and the `es6` target does not support native async/await, I prefer to target Node.js 8.x by default. Please note that this value should not be used by our build, but it may be used internally by VSCode's TypeScript plugin.